src/TypeSpec.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -91,127 +92,127 @@ export function mapTypeSpecTypeToCSharpInputTypeKind(
                 encode
             );
         case "ModelProperty":
-            return InputTypeKind.Object;
+            return InputPrimitiveTypeKind.Object;
```

Comment:
I would like to say this is none of this PR's business.
I also wonder do we really have object since it has been deprecated by tsp.

Reply:
well this might need to go through later as a clean up.
`object` in tsp is not represented by intrinsic types - it is a `model` without properties.

In this aspect, we could refer how TCGC does it - otherwise we get tons of troubles when adopting TCGC's models